docs/platform/what-is-tizen/versions/tizen-6-0-m2.md

```diff
@@ -0,0 +1,103 @@
+
+![img](media/6.0_1_web.png)
+
+### Lightweight Web Solution
+
+Escargot is the Tizen JavaScript engine for the product having hardware resource restriction. It uses about 14% in binary size and 35% in memory usage for the benchmark application compared to those of Chrome's V8 JavaScript engine. Escargot supports ES6 full specification and also supports ES 10 up to a large extent:
+
+ The following are the details for memory size and features in comparison to Chrome V8 benchmarks:
+
+  - Memory size  (vs chrome v8) octane benchmark in Ubuntu 18.04
+    - Binary size : 13.7 %
+    - Average RSS : 35.1 %
+
+  - Features
+    - Support ES6 Full feature and partial support of ES10
+
+## Machine Learning framework
+
+NNSteamer is an on-device AI framework that makes it easy to configure complex artificial neural networks in the application by making artificial neural network configurations with Gstreamer's plugin. 	 
+
+In Tizen 6.0, NN Runtime supports control-flow and dynamic tensor that enables voice model acceleration. In addition, by providing the NN Compiler front-end for NN Runtime, it can convert well-known models such as Tensorflow and TensorFlow-lite to be used on CE devices with insufficient resources on NN Runtime.	 
+
+- NNStreamer 	 
+ - Neural Network Pipeline. 	 
+   - .NET and C APIs supported. 	 
+   - Various accelerating hardware and frameworks supported. 	 	 
+   - Implement AI app more easily. 	 
+   - Execute AI app more efficiently. 	 
+
+- NNTrainer 	 
+ - On-Device Training. 	 
+   - Transfer learning, meta learning, and reinforcement learning. 	 
+   - APIs to construct and update neural network layers.
+
+- NN Runtime 	 
+ - Tizen’s default Neural Network inference framework (since v5.0).
+ - CPU and GPU mixed acceleration with profiling based optimal scheduling (since v5.5).
+ - Useful features for speech model support (in v6.0). 	 
+   - Control-flow, Dynamic Tensor, Shape Inference, etc. 	 
+
+- NN Compiler in Tizen SDK 	 
+ - Support well known ML frameworks, and model formats. 	 
+   - Tensorflow, Tensorflow-lite (preferential), Caffe, ONNX (experimental). 	 
+   - Easy scalability through Common IR and NN Package.
+
+
+   **Figure: NN Runtime**
+
+   ![NN Runtime](./media/6.0_1_NNRuntime.png)
+
+
+   **Figure: NN Streamer**
+
+   ![NN Streamer](./media/6.0_1_NNStreamer.png)
+
+### Core and Kernel
+- TIDL Extension, Multi-package installation supported in the Application framework.
+- Gesture FW and engine supported in Window and Interaction frameworks.
+- DALi (3D UI Toolkit) API set of DALi has been deprecated. DALi API set has been replaced with NUI (DALi C# layer) in Graphics frameworks
+- Ultra-wideband (UWB) ranging, Wi-Fi multiple interfaces are now supported in Network and Connectivity framework.
+- Kernel for Raspberry Pi 4 has been upgraded to version 5.4.50.
+- 64-Bit kernel and boot for Raspberry-Pi 4 has been supported.
+
+
```

Comment:
Kindly remove extra line space.

Reply:
I remove extra line space and the check the result. I think it seems to ok to me. please check it